src/liblx/util/util.cpp

```diff
@@ -35,27 +36,7 @@ safe_strdup(const char* s)
 
 
 
-LIBLX_EXTERN
-void* 
-safe_malloc (size_t size)
```

Comment:
we do not need these anymore?

Reply:
I moved the memory related ones to a file memory so that the include for these functions would be the same for liblx and libsbml